contracts/data/WitnetBoardData.sol

```diff
@@ -2,9 +2,55 @@
 
 pragma solidity >=0.7.0 <0.9.0;
 
+import "../exports/WitnetTypes.sol";
 
 /**
  * @title Witnet Board basal data model. 
  * @author Witnet Foundation
  */
-abstract contract WitnetBoardData {}
+abstract contract WitnetBoardData {  
+
+  struct SWitnetBoardData {
+    address owner;
+    address instance;
+    mapping (uint => SWitnetBoardDataRequest) requests;
+    uint256 noRequests;
```

Comment:
`noRequests` is a really bad name to indicate number of requests. Try `numRequests` or `requestsCount`.

Reply:
`numRequests` will be